bookkeeper-server/src/main/proto/BookkeeperProtocol.proto

```diff
@@ -62,6 +62,7 @@ enum OperationType {
     READ_LAC = 7;
     GET_BOOKIE_INFO = 8;
     START_TLS = 9;
+    SYNC = 10;
```

Comment:
FSYNC. SYNC could mean other things, like in zookeeper there's a sync op which has nothing to do with disks.

Reply:
But is maps to the 'sync()' function on the API

Reply:
sync() on which API? @eolivelli 
my issue is that sync is a very generic term, especially given that our system is a replicated log. it could mean may things.

Reply:
This is an example:
```java
try (WriteHandle wh...) {
     wh.append(data);
     wh.append(data);
     wh.sync();
     wh.append(data);
}
```

Reply:
Hmm, I think that sync() api should be renamed to force(). sync() sounds a little weak for this operation.